plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -319,6 +320,22 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->log_name_key = flb_sds_create(DEFAULT_LOG_NAME_KEY);
     }
 
+    tmp = flb_output_get_property("http_request_key", ins);
+    if (tmp) {
+        flb_sds_t http_request_key = flb_sds_create(tmp);
+        size_t http_request_key_size = flb_sds_len(http_request_key);
+        if (http_request_key_size < INT_MAX) {
+            ctx->http_request_key = http_request_key;
+            ctx->http_request_key_size = (int)http_request_key_size;
+        } else {
```

Comment:
```else``` must be in a new line :

https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions

Reply:
Fixed in https://github.com/fluent/fluent-bit/pull/4328/commits/1e4f8ae8f74d059b1c5802b83dfbadba30c9f404